The automatically generated patch puts this free form text on top of it.

If you are using gbp-buildpackage(1), you probably don't need this file.

If you are using dgit-maint-merge(7), please consider text as follows.

The Debian packaging of foo is maintained in git, using the merging workflow
described in dgit-maint-merge(7).  There isn't a patch queue that can be
represented as a quilt series.

A detailed breakdown of the changes is available from their canonical
representation - git commits in the packaging repository.  For example, to see
the changes made by the Debian maintainer in the first upload of upstream
version 1.2.3, you could use:

    % git clone https://git.dgit.debian.org/foo
    % cd foo
    % git log --oneline 1.2.3..debian/1.2.3-1 -- . ':!debian'

(If you have dgit, use `dgit clone foo`, rather than plain `git clone`.)

A single combined diff, containing all the changes, follows.
--- scratch-desktop-3.29.1.orig/electron-builder.yaml
+++ scratch-desktop-3.29.1/electron-builder.yaml
@@ -44,3 +44,5 @@ appx:
 nsis:
   oneClick: false # allow user to choose per-user or per-machine
   artifactName: "Scratch ${version} Setup.${ext}"
+linux:
+  target: dir
--- scratch-desktop-3.29.1.orig/scripts/electron-builder-wrapper.js
+++ scratch-desktop-3.29.1/scripts/electron-builder-wrapper.js
@@ -118,10 +118,10 @@ const calculateTargets = function (wrapp
             name: 'nsis:ia32',
             platform: 'win32'
         },
-	linuxDist: {
+        linuxDir: {
             name: 'dir',
             platform: 'linux'
-	}
+        }
     };
     const targets = [];
     switch (process.platform) {
@@ -150,8 +150,7 @@ const calculateTargets = function (wrapp
         targets.push(availableTargets.macDirectDownload);
         break;
     case 'linux':
-        // Run in two passes so we can skip signing the AppX for distribution through the MS Store.
-        targets.push(availableTargets.linuxDist);
+        targets.push(availableTargets.linuxDir);
         break;
     default:
         throw new Error(`Could not determine targets for platform: ${process.platform}`);
